aviary/subsystems/propulsion/engine_deck.py

```diff
@@ -742,10 +734,16 @@ def build_mission(self, num_nodes, aviary_inputs):
                           self.data[ELECTRIC_POWER],
                           units=units[ELECTRIC_POWER],
                           desc='Current electric energy rate (unscaled)')
+        # if self.use_nox:
         engine.add_output('nox_rate_unscaled',
                           self.data[NOX_RATE],
                           units=units[NOX_RATE],
                           desc='Current NOx emission rate (unscaled)')
+        # if self.use_shaft_power:
+        #     engine.add_output('shaft_power_unscaled',
+        #                       self.data[SHAFT_POWER_CORRECTED],
+        #                       units=units[SHAFT_POWER_CORRECTED],
+        #                       desc='Current corrected shaft power (unscaled)')
```

Comment:
question (non-blocking): Are you purposefully adding this here and leaving it commented for future use?

Reply:
I had originally added it there, Jason wanted engine deck to be generalized (not contain turboprop specific stuff), I wasn't sure if that would be valuable for accessory power, here we are